spec/rubocop/cop/style/guard_clause_spec.rb

```diff
@@ -132,14 +132,20 @@ def func
           test
         end
       end
+
+      def func
```

Comment:
I don't think that's the right place for this bit of code, given the description of the test. You can add a separate test for it.